alphagov/paas-cf
# [#136605385] Auto-deploy PaaS Roadmap

## What

We'd like to include roadmap deployment, in our production pipeline by default.

In any other environment, it should default to a false value, preventing it from being deployed without a need.

**Also, as part of this PR**

As part of #136550911 we will be replacing our CVE Notifier with a better approach.

We've decided, to remove the notifier sooner rather than later, in order to prevent it from spamming our support email.

## How to review

**Note:** It may prove itself to be difficult, if you're reviewing changes in the `Files changed` tab rather than separate `Commits`. Github may think I am replacing `cve-notifier` with `paas-dashboard`.

- Deploy from this branch, with `DEPLOY_ROADMAP=true` applied.
- Visit `https://paas-roadmap.${DEPLOY_ENV}.dev.cloudpipelineapps.digital/` and see if everything in order.

**Optionally extra checks**

- Deploy from this branch, with `ENABLE_CVE_NOTIFIER=true` applied.
- After running `cf apps`, make sure `cve-notifier` app does not exist.

## Who can review

Anyone but @paroxp
